plugins/node/opentelemetry-koa-instrumentation/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@opentelemetry/koa-instrumentation",
+  "version": "0.8.0",
+  "description": "OpenTelemetry Koa automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "koa",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "plugin",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@koa/router": "^9.3.1",
+    "@opentelemetry/context-async-hooks": "0.9.0",
```

Comment:
opentelemetry dependencies out of date

Reply:
Please bump core deps to 0.12.2

Reply:
@dyladan I would love it if you could help me look into this if you get a chance - I'm getting errors from these new versions of the opentelemetry core modules across other plugins as well as my own while developing locally. I also tried recloning a fresh copy of the contrib repo, then testing other plugins such as Express and MongoDB and ran into these same errors. I sent more information and repro steps over Gitter and would really appreciate it if you could take a look